sanowret-crystal.lic

```diff
@@ -14,6 +14,7 @@ class SanowretCrystal
     @invalid_rooms = []
     @worn_crystal = false
     args = parse_args(arg_definitions)
+    @adjective = get_settings.sanowret_adjective ? get_settings.sanowret_adjective : 'sanowret'
```

Comment:
💡 It's minor, but this script now loads settings from your YAML three times (twice on this line, once on line 18). I recommend saving it off to a local var.